apps/src/music/player/MusicPlayer.ts

```diff
@@ -162,6 +178,45 @@ export default class MusicPlayer {
     }
   }
 
+  playPatternAtMeasureById(
+    value: any,
```

Comment:
this type can be `PatternEvent["value"]` now (could also break out the `value` type into its own interface if you'd like)

Reply:
Ah yes, gotta add `any` to the linter! :)